pravega-k8-log-collection/pravega-k8s-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+mkdir -p /tmp/logdir
```

Comment:
Use the shell flags `set -euo pipefail` for safety. More context here: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
+1
Sure will update.